bin/lib/validate-url.js

```diff
@@ -0,0 +1,58 @@
+const debug = require('debug')('autovoice:validate-url');
+
+// copied from https://gist.github.com/dperini/729294
+//    with added check for localhost
+// via https://mathiasbynens.be/demo/url-regex
+
+const RE_WEBURL = new RegExp(
+  "^" +
+    // protocol identifier
+    "(?:(?:https?|ftp)://)" +
```

Comment:
I just copied in the regex which appeared to be the best overall at correctly recognising valid urls. Easy enough to strip the ftp part out, but I wanted to leave it alone as far as poss.